Dockerfile.brooklyn

```diff
@@ -17,7 +17,7 @@ WORKDIR /usr/src/app
 
 COPY  . .
 
-RUN cargo build --locked --release --config net.git-fetch-with-cli=true
```

Comment:
let's keep the --locked flag & not remove it from the builds.

Reply:
done